theme/theme.scss

```diff
@@ -0,0 +1,17 @@
+figure {
```

Comment:
What I mean is that we'll need "widget styles" and its the widget container what should be styled. So e.g. you shouldn't be styling the `img` below because the widget system won't know about content of widgets.